plugins/generator-1.20.1/forge-1.20.1/templates/item/item.java.ftl

```diff
@@ -273,10 +273,24 @@ public class ${name}Item extends Item {
 						double y = entity.getY();
 						double z = entity.getZ();
 						if (<@procedureOBJToConditionCode data.rangedUseCondition/>) {
+							<#if data.chargesPower>
+								float pullingPower = BowItem.getPowerForTime(this.getUseDuration(itemstack) - time);
+								if (!((double) pullingPower < 0.1)) {

```

Comment:
if it's not smaller than 0.1, it's greater or equal to 0.1... maybe you should simply write `pullingPower >= 0.1`

Reply:
I just used what BowItem uses

Reply:
I agree this "if" could be made cleaner

Reply:
> I just used what BowItem uses

(Mojang and decisions making senses are not compatible elements)

Reply:
It may be caused by the decompiler so let's give it a benefit of a doubt

Reply:
So weird compiler it is